BenchTesting.java

```diff
@@ -62,5 +65,87 @@ public static void main(String[] args) {
         System.out.println("Expected: temp1. Actual: " + island1.getPassDetails(123));
         System.out.println("Expected: true. Actual: " + island1.isPassOnIsland(pass1));
         System.out.println("Expected: false. Actual: " + island1.isPassOnIsland(pass2));
+//Testing for Ferry class-> To test everything for this class, you need to comment the Island and Pass variables above ^
+
+
+
+//        Island island1 = new Island(111, "Haven", 5, 2);
+//        Island island2 = new Island(222, "Sweetfish", 3, 2);
+//        Island island3 = new Island(333, "Seahorse", 2, 1);
+//        Island island4 = new Island(442, "Egg", 10, 2);
+//
+//
+//        Pass pass1 = new Pass(123, "Temp1", 5, 10, 1);
+//        Pass pass2 = new Pass(456, "Temp2", 3, 20, 6);
+//        Pass pass3 = new Pass(789, "Temp3", 8, 5, 0);
+//        Pass pass4 = new Pass(111, "Temp4", 10, 2, 3);
+
+
+
+        Ferry ferry1 = new Ferry(1,island1,island2);
+        Ferry ferry2 = new Ferry(2,island2,island3);
+        Ferry ferry3 = new Ferry(3,island3,island4);
+//  Multiple tests
+        island1.enter(pass1);
+        ferry1.ProcessPass(pass1,island1,island2);
+
+
+        System.out.println("Expected: 1. Actual: " +ferry1.getFerryCode());
+        System.out.println("Expected: island2(Sweetfish). Actual: " +ferry2.getSourceIsland());
+        System.out.println("Expected: island4(Egg). Actual: " +ferry3.getDestinationIsland());
+
+        System.out.println("Expected: true. Actual:\n " +ferry1.CanPassTravel(pass1));
+        System.out.println("Expected: false. Actual:\n " +ferry2.CanPassTravel(pass4));
+
+
+
+
+
+//        island1.enter(pass1);
```

Comment:
Why is there alot of commented code?

Reply:
i did lots of testing yesterday, to be sure it was correct